src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,37 +1,156 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
 import { TextField } from '../TextField';
+import { NewMovieProps } from '../../types/Movie';
 
-export const NewMovie = () => {
+export const NewMovie: React.FC<NewMovieProps> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
```

Comment:
Create only one state for movie
```suggestion
  const [movie, setMovie] = useState({
    movie: '',
    ........
  });
```